tribe-ext-settings-import-export.php

```diff
@@ -0,0 +1,328 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Settings Import / Export
+ * Plugin URI:        https://theeventscalendar.com/extensions/settings-import-export/
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-settings-import-export
+ * Description:       You can import and export the settings of The Events Calendar
+ * Version:           1.0.0
+ * Extension Class:   Tribe\Extensions\Settings_Import_Export\Main
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       tribe-ext-settings-import-export
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+namespace Tribe\Extensions\Settings_Import_Export;
+
+use Tribe__Extension;
+
+/**
+ * Define Constants
+ */
+
+if ( ! defined( __NAMESPACE__ . '\NS' ) ) {
+	define( __NAMESPACE__ . '\NS', __NAMESPACE__ . '\\' );
+}
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+	class_exists( 'Tribe__Extension' )
+	&& ! class_exists( NS . 'Main' )
+) {
+	/**
+	 * Extension main class, class begins loading on init() function.
+	 */
+	class Main extends Tribe__Extension {
+
+		/**
+		 * Extension initialization and hooks.
+		 */
+		public function init() {
+			// Load plugin textdomain
+			load_plugin_textdomain( 'tribe-ext-settings-import-export', false, basename( dirname( __FILE__ ) ) . '/languages/' );
```

Comment:
Totally optional, but since we're no longer writing PHP 5.2 compatible code we could simplify this line:
```suggestion
			load_plugin_textdomain( 'tribe-ext-settings-import-export', false, basename( __DIR__ ) . '/languages/' );
```